tests/EventListener/InitializeSystemListenerTest.php

```diff
@@ -157,6 +157,58 @@ public function testRequestWithoutScope()
     }
 
     /**
+     * Tests a request without scope.
+     *
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     *
+     * @expectedException \Contao\CoreBundle\Exception\InsecureInstallationHttpException
+     */
+    public function testValidateInstallation()
```

Comment:
As it expects the `InsecureInstallationHttpException` to be thrown which is part of my PR, yes it is related?